Wordle.Api/Controllers/DateWordController.cs

```diff
@@ -19,6 +19,21 @@ public DateWordController(GameService gameService)
     {
         _gameService = gameService;
     }
+    [HttpGet]
+    public IEnumerable<DateWordDto> Get(string playerGuid = "Fine if not a Guid")
```

Comment:
I think you want your get to just return a single daily word right? Not a list of them.